stencil-library/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ### Changelog
 
+### 4.6.5
```

Comment:
I don't think there's any need to update the changelog since the storybook documentation is not distributed with the package (doesn't change production code)